src/FixedDataTable.js

```diff
@@ -932,25 +953,23 @@ class FixedDataTable extends React.Component {
       reorderColumn: columnKey,
     });
 
-    if (scrollStart !== scrollX && onHorizontalScroll) {
-      onHorizontalScroll(scrollX)
+    if (scrollStart !== scrollX) {
+      this._scrollToX(scrollX)
     };
   }
 
   _onScroll = (/*number*/ deltaX, /*number*/ deltaY) => {
     const {
       maxScrollX,
       maxScrollY,
-      onHorizontalScroll,
-      onVerticalScroll,
-      scrollActions,
       scrollFlags,
       scrollX,
       scrollY,
       scrolling,
     } = this.props;
     const { overflowX, overflowY } = scrollFlags;
 
+    this._didScrollStop.reset();
```

Comment:
Is there any reason to reset this early on? Also wouldn't the this_.scrollToX/this._scrollToY calls below call this._onHorizontalScroll/this._onVerticalScroll which would in turn reset it again?

Reply:
because this function will always trigger stop again, so resetting existing one to avoid unnecessary renders bcs of  `isScrolling false flag` when scroll is done in quick successions.

Reply:
> this function will always trigger stop again
Yes, then doesn't it mean that the stop call is debounced again, thus avoiding the need of the reset call?

Reply:
ok, got your concern here, will make changes!